### PR TITLE
mysqlcluster: Debug Mode for Pod #375

### LINKED ISSUE #375: [feature] Debug Mode for Pod
**Is your feature request related to a problem? Please describe.**

When MySQL crash or other fault, mysql container in Pod will restart , it will clear the logs and coredump file, so make it everything bad.

**Describe the solution you'd like**
use debug mode for mysql container in pod,  it should be never restart the container.

**Describe alternatives you've considered**

**Additional context**

### PR DESCRIPTION
**Pull request by acekingke**
### What type of PR is this?

/enhancement
### Which issue(s) this PR fixes?

Fixes #375 

### What this PR does?

Summary:
add the debug mode, make container never recreate when mysqld crashed

### CODE REVIEW

File: docs/en-us/DebugMode.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Debug Mode
+When you want avoid the restart-on-fail loop for mysql container, You should use Debug Mode.
+it just use create a empty file  `/var/lib/mysql/sleep-forever`
+for example:
+```bash
+kubectl exec -it sample-mysql-0 -c mysql -- sh -c 'touch /var/lib/mysql/sleep-forever'
+```
+it make pod sample-mysql-0's mysql container will never restart when mysqld is crashed.
```
Comment (andyli029) [inline docs/en-us/DebugMode.md:8]: Add cmd about cancel Debug

**Comment by andyli029** [comment]
Ping @runkecheng

**Review by runkecheng — CHANGES_REQUESTED** [review]

**Review by runkecheng — APPROVED** [review]
lgtm

**Comment by runkecheng** [comment]
I am not sure that there will be other risks in the container if run the main logic not in the first process.

**Review by andyli029 — APPROVED** [review]

**Pull request merged**